webcompat/templates/error.html

```diff
@@ -1,13 +1,13 @@
 {% extends "layout.html" %}
 {% block body %}
 {% include "shared/nav.html" %}
-<div class="wc-UISection">
-  <main class="wc-UIContent" id="pageerror" role="main">
-    <h1 class="wc-Title wc-Title--l">Yikes!  {{ error_message }} ({{ error_code }})</h1>
-    <h2 class="wc-Title wc-Title--m">(sound of broken glass)</h2>
+<article class="grid-row">
+	<div class="grid-cell x3">
```

Comment:
As mentioned in the other pull request, this repo uses two spaces instead of tabs for indentation.